src/generate/interrupt.rs

```diff
@@ -136,7 +118,7 @@ pub fn render(
                 #[link_section = ".vector_table.interrupts"]
                 #[no_mangle]
                 #[used]
-                pub static INTERRUPTS:
+                pub static __INTERRUPTS:
```

Comment:
How will this affect other architectures?

Reply:
I changed the name for parity with `CortexM`. It doesn't affect other architectures because the match arm only runs when the arch is `Msp430`. The name should've been changed a while ago, but I only got around to it now.